src/main/webapp/script.js

```diff
@@ -97,4 +97,12 @@ async function image_upload() {
   }
 }
 
+function changeDescription(imgSrc, txt){
```

Comment:
Does this function change the image description after the image is uploaded?

Reply:
It changes the description and image each time you click on an image, since the popup is just like an empty template

Reply:
I see nice